### PR TITLE
If the draft course is in Reviewed, both draft and official should be kicked back to Unpublished

### PR DESCRIPTION
**Pull request by inventhouse**

### CODE REVIEW

**Review by Dillon-Dumesnil — APPROVED** [review]
LGTM

File: course_discovery/apps/api/v1/views/course_runs.py [hunk 1]
```diff
@@ -216,6 +216,9 @@ def update(self, request, *args, **kwargs):
         # revert status to unpublished.
         if course_run.status == CourseRunStatus.Reviewed:
             request.data['status'] = CourseRunStatus.Unpublished
+            official_run = course_run.official_version
+            official_run.status = CourseRunStatus.Unpublished
```
Comment (dianakhuang) [inline course_discovery/apps/api/v1/views/course_runs.py:221]: Ww should be assured that an official version exists at this point, but should we be defensive about it anyway?

**Review by dianakhuang — APPROVED** [review]
Minor nitpick.

File: course_discovery/apps/api/v1/views/course_runs.py (hunk 1 above)
Comment (inventhouse) [reply course_discovery/apps/api/v1/views/course_runs.py:221]: Something like `if not official_run: official_run = ensure_official_version()`?

Comment (dianakhuang) [reply course_discovery/apps/api/v1/views/course_runs.py:221]: Yeah, I think that would be a good idea, just to be defensive about it.

**Pull request merged**